llvm/lib/Support/xxhash.cpp

```diff
@@ -47,6 +47,19 @@
 
 #include <stdlib.h>
 
+#if !defined(LLVM_XXH_USE_NEON)
+#if (defined(__aarch64__) || defined(_M_ARM64) || defined(_M_ARM64EC)) &&      \
```

Comment:
`clang --target=aarch64-windows-msvc -dM -xc /dev/null -E ` also defines `__aarch64__`, so the Windows specific macros might not be useful?

Reply:
MSVC proper does not define `__aarch64__`, though: https://godbolt.org/z/qhY1x4P4G. But the NEON intrinsics this code uses work there, too.